src/scss/layout/_mobile-menu.scss

```diff
@@ -0,0 +1,27 @@
+@import '../base/variables';
+
+.mobile-menu {
```

Comment:
- [optional] please ensure that when the mobile popup comes up it should not be possible to scroll through your site. great job